src/pages/blog/_components/ArticleCard.astro

```diff
@@ -10,6 +11,7 @@ interface Props {
 }
 
 const { post } = Astro.props;
+const { remarkPluginFrontmatter } = await render(post);
```

Comment:
I don't like how this is currently only available via the `render()` method since we're technically doing it twice now. 

I couldn't find a way to share data between this page and the other one but might be missing something obvious there, advice appreciated!